### PR TITLE
SI-8503 -version is info setting 

### PR DESCRIPTION
**Pull request by som-snytt**
And the Scala runner exits with 0 for info settings.

```
apm@mara:~$ javac -version 2> /dev/null
apm@mara:~$ echo $?
0
apm@mara:~$ java -version 2> /dev/null
apm@mara:~$ echo $?
0
apm@mara:~$ java -help 2> /dev/null
apm@mara:~$ echo $?
0
apm@mara:~$ java -junk 2> /dev/null
apm@mara:~$ echo $?
1
apm@mara:~$ skala -version
Scala code runner version 2.11.2-20140528-130424-c8f8782f6a -- Copyright 2002-2013, LAMP/EPFL
apm@mara:~$ echo $?
0
apm@mara:~$ skalac -version
Scala compiler version 2.11.2-20140528-130424-c8f8782f6a -- Copyright 2002-2013, LAMP/EPFL
apm@mara:~$ echo $?
0
apm@mara:~$ skalac -junk 2> /dev/null
apm@mara:~$ echo $?
1
```

Review by @adriaanm in case this breaks one of his scripts

### CODE REVIEW

**Comment by som-snytt** [comment]
PLS REBUILD/pr-scala@c8f8782

**Comment by adriaanm** [comment]
PLS SYNCH

File: src/compiler/scala/tools/nsc/GenericRunnerCommand.scala [hunk 1]
```diff
@@ -86,8 +86,7 @@ A file argument will be run as a scala script unless it contains only
 self-contained compilation units (classes and objects) and exactly one
 runnable main method.  In that case the file will be compiled and the
 main method invoked.  This provides a bridge between scripts and standard
-scala source.
-  """) + "\n"
+scala source.%n"""
```
Comment (adriaanm) [inline src/compiler/scala/tools/nsc/GenericRunnerCommand.scala:89]: whence the percentage?

Comment (adriaanm) [inline src/compiler/scala/tools/nsc/GenericRunnerCommand.scala:89]: ah, the repl tells me %n and \n are twins. Did not know.

**Comment by adriaanm** [comment]
LGTM. Thanks for sweeping up around the block.

File: src/compiler/scala/tools/nsc/GenericRunnerCommand.scala (hunk 1 above)
Comment (som-snytt) [inline src/compiler/scala/tools/nsc/GenericRunnerCommand.scala:89]: Sometimes %n is \r\n.

**Pull request merged**